IS/Sem.06/Pract.06/Task05.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+
+int powOfTwo(int num)
+{
+	int result = 1;
+	for (int i = 0; i < num; i++)
+	{
+		result *= 2;
+	}
+	return result;
+}
+
+int theResult(int num, int k)
+{
+	int result = 0, copyOfK = k;
+	while (k != 0)
+	{
+		if (num % 2 == 0)
+		{
+			num /= 2;
+			--k;
+		}
+		else
+		{
+			result += powOfTwo(copyOfK - k);
+			num /= 2;
+			--k;
+		}
+	}
+	return result;
+}
```

Comment:
The point of the task is to use bitwise operations
Think of a better name for this function